pkg/pillar/hypervisor/kvm.go

```diff
@@ -674,24 +674,28 @@ func (ctx kvmContext) CreateDomConfig(domainName string, config types.DomainConf
 	return nil
 }
 
-func waitForQmp(domainName string) error {
+func waitForQmp(domainName string, absent bool) error {
```

Comment:
`absent` is really not clear as a variable name here. Can we come up with something more descriptive?